packages/core-products/db/product-media/helpers.js

```diff
@@ -8,6 +8,8 @@ ProductMedia.findProductMedia = ({ productMediaId }) => {
 };
 
 ProductMedia.removeProductMedia = ({ productMediaId }) => {
+  const productMedia = ProductMedia.findProductMedia({ productMediaId });
+  Media.removeFiles({ _id: productMedia.mediaId });
```

Comment:
Not good, removing a product media does not automatically mean to remove the media, media's can be re-used